### ISSUE TITLE
DoesNotExistException Error

### ISSUE THREAD

**Issue by numerfolt** [owncloud/music#1071]
### Steps to reproduce
1. Use play:sub with bookmarks enabled (for resuming titles)
2. Play some music
3. Look for error message in nextcloud logfile

### Expected behaviour
There should be no errors

### Actual behaviour
There are quite some Errors like this:
```
OCP\AppFramework\Db\DoesNotExistException: Did expect one result but found none when executing: query "SELECT * FROM `*PREFIX*music_bookmarks` WHERE `*PREFIX*music_bookmarks`.`user_id` = ? AND `type` = ? AND `entry_id` = ? "; parameters Array ( [0] => ***REMOVED USERNAME*** [1] => 1 [2] => 70229 ) ; limit ""; offset ""
```

### Server configuration
**Operating system**: Linux nextcloudpi 6.1.21-v8+

**Web server:** Server version: Apache/2.4.56 (Debian)

**Database:** 10.5.19-MariaDB-0+deb11u2 Debian 11

**PHP version:** PHP 8.1.18

**Nextcloud version:** 26.0.1.1

**The content of config/config.php:**
https://gist.github.com/numerfolt/22bb1515250bee113e606890d16da1d6

**List of activated apps:**

```
Enabled:
  - activity: 2.18.0
  - bookmarks: 13.0.1
  - calendar: 4.3.4
  - circles: 26.0.0
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contacts: 5.2.0
  - contactsinteraction: 1.7.0
  - cookbook: 0.10.2
  - cospend: 1.5.10
  - dashboard: 7.6.0
  - dav: 1.25.0
  - federatedfilesharing: 1.16.0
  - files: 1.21.1
  - files_pdfviewer: 2.7.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - firstrunwizard: 2.15.0
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - maps: 1.0.2
  - music: 1.8.4
  - nextcloud_announcements: 1.15.0
  - nextcloudpi: 0.0.1
  - notes: 4.7.2
  - notifications: 2.14.0
  - notify_push: 0.6.3
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - photos: 2.2.0
  - previewgenerator: 99.99.99
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - recommendations: 1.5.0
  - related_resources: 1.1.0-alpha1
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - sharebymail: 1.16.0
  - side_menu: 3.8.0
  - survey_client: 1.14.0
  - systemtags: 1.16.0
  - tasks: 0.15.0
  - text: 3.7.2
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - updatenotification: 1.16.0
  - user_status: 1.6.0
  - viewer: 1.10.0
  - weather_status: 1.6.0
  - workflow_ocr: 1.26.1
  - workflowengine: 2.8.0
Disabled:
  - admin_audit: 1.16.0
  - bruteforcesettings: 2.6.0
  - encryption: 2.14.0
  - federation: 1.16.0 (installed 1.16.0)
  - files_external: 1.18.0
  - memories: 5.1.0 (installed 5.1.0)
  - news: 21.2.0 (installed 21.2.0)
  - support: 1.9.0 (installed 1.9.0)
  - suspicious_login: 4.4.0
  - tables: 0.5.0 (installed 0.5.0)
  - twofactor_totp: 8.0.0-alpha.0
  - user_ldap: 1.16.0
```

#### Web server error log
https://gist.github.com/numerfolt/e0dec3f1a60b3aeab1c7ea66bc14d641

#### Nextcloud log
https://gist.github.com/numerfolt/5ae9352a329249d5a20b7a617476eb31

**Comment by paulijar** [comment]
Thanks for the report. This seems to happen when a Subsonic client attempts to delete a bookmark which doesn't exist. I don't know why your client does that but the Music app should handle this case more gracefully.

**Comment by paulijar** [comment]
The newly released Music v1.9.1 should no longer log these errors.
